RM_SUI/RM_SUI/View/GreetingView.swift

```diff
@@ -0,0 +1,120 @@
+        static let signInButtonName = "Sign in here"
+        static let mainSpacerMinLength: CGFloat = 50
+        static let noAccountTextPadding: CGFloat = 30
+        static let logoImageFrameWidthHeight: CGFloat = 250
+        static let getStartedButtonPadding: CGFloat = 20
+        static let signButtonTextOffset: CGFloat = 15
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        NavigationView {
+            ZStack {
+                backgroundGradientView
+                VStack {
+                    storeNameTextView
+                    logoImageView
+                    Spacer(minLength: Constants.mainSpacerMinLength)
+                    getStartedButtonView
+                    Text(Constants.noAccountMessage)
+                        .font(.headline)
+                        .fontWeight(.bold)
+                        .padding(.top, Constants.noAccountTextPadding)
+                    sightInButton
+                }
+            }
+            .foregroundColor(Color(.systemBackground))
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @StateObject var greetingViewModel: GreetingViewModel
+
+    private var backgroundGradientView: some View {
+        LinearGradient(
+            colors: Constants.backgroundGradient,
+            startPoint: .top,
+            endPoint: .bottom
+        )
+        .ignoresSafeArea(.all)
+    }
+
+    private var storeNameTextView: some View {
+        Text(Constants.storeName)
+            .font(.largeTitle)
+            .fontWeight(.bold)
+            .multilineTextAlignment(.center)
+            .lineSpacing(Constants.lineSpacing)
+            .padding(Constants.storeTextPadding)
+    }
+
+    private var logoImageView: some View {
+        AsyncImage(url: greetingViewModel.logoImageURL) { image in
+            image
+                .resizable()
+                .frame(
+                    width: Constants.logoImageFrameWidthHeight,
+                    height: Constants.logoImageFrameWidthHeight
+                )
+        } placeholder: {
+            ProgressView()
+        }
+    }
+
+    private var getStartedButtonView: some View {
+        NavigationLink {
+            ProductInfoView()
+                .navigationBarBackButtonHidden()
+        } label: {
+            Text(Constants.buttonGetStartedName)
+                .font(.largeTitle)
+                .fontWeight(.bold)
+                .foregroundStyle(Constants.buttonGradient)
+        }
+        .padding(Constants.getStartedButtonPadding)
+        .background(Capsule())
+    }
+
+    private var sightInButton: some View {
+        NavigationLink {
+            SignInView()
+                .navigationBarBackButtonHidden()
+        } label: {
+            Text(Constants.signInButtonName)
+                .font(.largeTitle)
+                .fontWeight(.bold)
+                .underline()
+                .baselineOffset(Constants.signButtonTextOffset)
+        }
+    }
+}
+
+/// Preview
```

Comment:
```suggestion
/// Preview
```
Видел что Женя многим удалял описание для PreviewProvider, возможно оно и не нужно, лучше у него уточнить